secretballot/middleware.py

```diff
@@ -1,11 +1,17 @@
 # -*- coding: utf-8 -*-
 from hashlib import md5
 
-# for support Django 1.10+
-from django.utils.deprecation import MiddlewareMixin
+from django import VERSION
 
+a, b = VERSION[:2]
+if a < 1 or a == 1 and b < 10:
```

Comment:
Then Pre-1.0 releases do not enter the condition (0.95, 0.96).
Suggest the following option:
`if (a < 1) or (a == 1 and b < 10):`